README.md

```diff
@@ -28,18 +28,17 @@
 
 ## Installation
 
-There are a few ways to install:
+Follow our [Quick Start guide][quick-start].
 
-* gulp v4.0.0 - `npm install gulp@next`
-* gulp v4.0.0-alpha.3 - `npm install gulpjs/gulp#v4.0.0-alpha.3`
-* gulp v3.9.1 - `npm install gulp`
+## Roadmap
+
+Find out about all our work-in-progress and outstanding issues at https://github.com/orgs/gulpjs/projects.
 
 ## Documentation
 
-For a Getting started guide, API docs, recipes, making a plugin, etc. check out our docs!
+Check out the [Getting Started guide][getting-started-guide] and [API docs][api-docs] on our website!
```

Comment:
For consistency with the link above and title of the page, you could replace `Getting Started` by `Quick Start`.

Reply:
They are slightly difference references so I'm going to stick with this.